responsible-ai-collaborative/aiid
# Add `onIncidentUpdate` trigger and function

Closes #1142

## Testing

Preview link for testing: https://deploy-preview-8--pablocosta-staging.netlify.app

1. Login and subscribe to an incident. ie: https://deploy-preview-8--pablocosta-staging.netlify.app/cite/11
2. Login on another browser with another Admin user
3. Go to https://deploy-preview-8--pablocosta-staging.netlify.app/apps/submitted and promote a report to the incident selected in step # 1
4. The new trigger should be executed and the user in step # 1 should receive the Incident update email notification
